util/extensions.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs')
+
+const pathFromRoot = require('../util/pathFromRoot')
+const packageJson = require('../package.json')
+
+const getDependencyConfig = dependencyName => pathFromRoot('node_modules', dependencyName, 'govuk-prototype-kit.config.json')
```

Comment:
Should this be `getDependencyConfigFilename`?

Reply:
Negotiated to 'getDependencyConfigPath'